presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -0,0 +1,284 @@
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create jvm.properties and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    static Path createCoordinatorLogProperties()
+    {
+        try {
+            Path tempConfigProperties = Files.createTempFile("log", ".properties");
+            String properties = "com.facebook.presto=DEBUG\n";
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create log.properties and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    public static Path createCoordinatorNodeProperties()
+    {
+        try {
+            Path tempConfigProperties = Files.createTempFile("node", ".properties");
+
+            String properties = "node.environment=testing\n" +
+                    "node.location=testing-location\n" +
+                    "node.data-dir=/var/lib/presto/data\n";
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create config.properties and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    public static Path createNativeWorkerEntryPointScript()
+            throws IOException
+    {
+        Path entryPointScript = Files.createTempFile("entrypoint", ".sh");
+
+        String scriptContent =
+                "#!/bin/sh\n\n" +
+                        "GLOG_logtostderr=1 presto_server \\\n" +
+                        "    --etc-dir=/opt/presto-server/etc\n";
+        Files.write(entryPointScript, scriptContent.getBytes(), StandardOpenOption.WRITE);
+
+        entryPointScript.toFile().setExecutable(true);
+        return entryPointScript;
+    }
+
+    public static Path createCoordinatorEntryPointScript()
+    {
+        try {
+            Path entryPointScript = Files.createTempFile("entrypoint", ".sh");
+
+            String scriptContent =
+                    "#!/bin/sh\n" +
+                            "set -e\n" +
+                            "$PRESTO_HOME/bin/launcher run\n";
+            Files.write(entryPointScript, scriptContent.getBytes(), StandardOpenOption.WRITE);
+
+            entryPointScript.toFile().setExecutable(true);
+            return entryPointScript;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create entrypoint.sh and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    static Path createNodeProperties(String nodeId)
```

Comment:
Call this createWorkerNodeProperties to be consistent with the rest of the naming in this class.

Reply:
Modified the code.